packages/dialect-react-ui/components/Notifications/index.tsx

```diff
@@ -85,38 +90,52 @@ function CreateThread() {
   );
 }
 
-function Settings(props: { toggleSettings: () => void }) {
+function Settings(props: {
+  toggleSettings: () => void;
+  notifications: NotificationType[];
+}) {
   const { dialectAddress, deleteDialect, isDialectDeleting, deletionError } =
     useDialect();
   const { colors, textStyles, icons } = useTheme();
 
   return (
     <>
       <div className="mb-3">
-        <p className={cs(textStyles.body, 'mb-1')}>Included event types</p>
-        <ul className={cs(textStyles.bigText, 'list-disc pl-6')}>
-          <li>Welcome message on thread creation</li>
-        </ul>
+        <h2 className={cs(textStyles.h2, 'mb-1')}>Notifications</h2>
```

Comment:
We should probably have a "no notifications supplied" or some other kind of text if none are passed, what do you think?

Reply:
I wonder if it make sense for dev of dapp to leave these empty?